components/molecules/SonificationControls/index.tsx

```diff
@@ -0,0 +1,27 @@
+"use client";
+import { FC } from "react";
+import AladinOverlay from "@/components/atomic/AladinOverlay";
+import Zoom from "../Controls/Zoom";
+import ControlStack from "../Controls/Stack";
+import PlayPauseButton from "./PlayPauseButton/index";
+import styles from "./styles.module.css";
```

Comment:
build tool got mad that this was declared and not used, just need to delete the import